vale/styles/jargonLint/Licenses.yml

```diff
@@ -21,16 +21,19 @@ swap:
   EPL[ -]?1(?:\.0)?: EPL 1.0
   EPL[ -]?2(?:\.0)?: EPL 2.0
   EUPL: EUPL
-  EUPL[ -]?v?1(?:\.2)?: EUPL v1.2
+  EUPL[ -]?v?1\.1: EUPL v1.1
+  EUPL[ -]?v?1\.2: EUPL v1.2
   ISC: ISC
   MIT: MIT
   MirOS: MirOS
   MPL: MPL
+  MPL[ -]?1(?:\.1)?: MPL 1.1
   MPL[ -]?2(?:\.0)?: MPL 2.0
   OFL: OFL
+  OFL[ -]?1(?:\.1)?: OFL 1.1
```

Comment:
We have a problem here. The rules don't correct capitalization mistakes regarding "OFL 1.1".

The rule corrects ``ofl`` but avoids ``ofl 1.1``.

OFL is not a very popular license, but the same issue is relevant for other licenses in Licenses.yml such as MPL and EPL. I only pointed the issue in OFL to ease testing. Similar to our discussions with GPLv2 the fix can be replicated.

Reply:
This issue is likely something that I need to clarify with the upstream. I'm resolving this and merging this pull request for now, since you've also approved it.